README.md

```diff
@@ -61,6 +61,26 @@ You can now push to the specified environment by running
 skaffold run
 ```
 
+## Connecting psql client using the Cloud SQL Proxy:
+Follow [Installing this instruction](https://cloud.google.com/sql/docs/mysql/sql-proxy#macos-64-bit)
+to install Cloud SQL Proxy 
+
+Go to cloud console to get the instance name you want to connect to, then start the proxy:
+```
+./cloud_sql_proxy -instances=<INSTANCE_CONNECTION_NAME>=tcp:5432
+```
+Start the client session
+```
+psql "host=127.0.0.1 sslmode=disable dbname=<DB_NAME> user=<USER_NAME>"
+```
+For Broad engineer, DB_NAME and USER_NAME can be found in vault. 
```

Comment:
add that the instance name can be found under `...crl_janitor/postgres/instance` in vault?

Reply:
done